data-serving/data-service/src/controllers/case.ts

```diff
@@ -16,30 +17,43 @@ export const get = (req: Request, res: Response): void => {
  * Handles HTTP GET /cases.
  */
 export const list = (req: Request, res: Response): void => {
-    res.send('Triggered list cases.');
+    Case.find({}, (err, c) => {
+        if (err) {
+            res.status(500).send(err);
+        }
+        res.json(c);
+    });
 };
 
 /**
  * Create a case.
  *
  * Handles HTTP POST /cases.
- *
- * For now, just attempts to parse an "age" field and return it in the response.
- * Returns 422 if "age" isn't present in the request body.
  */
 export const create = async (req: Request, res: Response): Promise<void> => {
-    await check('age', 'Age must be a valid number')
-        .not()
-        .isEmpty()
-        .isNumeric()
+    await check('outcome', `Outcome must be one of: ${validOutcomes}`)
```

Comment:
might be nice to (eventually) move this validation logic into the the model, in some sort of static #fromJSON method, so all of the validation is in one place

Reply:
+1 -- we'll have to mess around with this a bit.